Lib/fontTools/otlLib/maxContextCalc.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import print_function, division, absolute_import, unicode_literals
+
+__all__ = ['maxCtxFont']
+
+
+def maxCtxFont(font):
+    """Calculate the usMaxContext value for an entire font."""
+
+    maxCtx = 0
+    for tag in ('GSUB', 'GPOS'):
+        if tag not in font:
+            continue
+        table = font[tag].table
+        if getattr(table, 'LookupList', None) is None:
```

Comment:
```suggestion
        if not table.LookupList:
```

the attribute is guaranteed to be there (as of current master)

Reply:
sorry, copied the wrong file. Thanks for catching it